cmd/cdi-controller/controller.go

```diff
@@ -142,6 +161,10 @@ func start(cfg *rest.Config, stopCh <-chan struct{}) {
 	go pvcInformerFactory.Start(stopCh)
 	go podInformerFactory.Start(stopCh)
 	go serviceInformerFactory.Start(stopCh)
+	go ingressInformerFactory.Start(stopCh)
```

Comment:
Do we always need to start this? even if it's openshift and we use routeInformerFactory?

Reply:
yes, we support both ingress and route on openshift